examples/userapi/07_functions_on_subdomains.ipynb

```diff
@@ -0,0 +1,3028 @@
+{
```

Comment:
<p>that's a nice one!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
<p>Thanks!</p>